contrib/devtools/install-golangci-lint.sh

```diff
@@ -2,20 +2,25 @@
 
 set -euo pipefail
 
+f_sha256() {
+  local l_file
+  l_file=$1
+  python -sBc "import hashlib;print hashlib.sha256(open('$l_file','rb').read()).hexdigest()"
```

Comment:
this py2 print might result in issues if python 3 is the default
maybe adding `from __future__ import print_function` would be useful

Reply:
Fixed, can you review it again please?